Gemfile

```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.2'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 6.0.3', '>= 6.0.3.4'
+# Use postgresql as the database for Active Record
+gem 'pg', '>= 0.18', '< 2.0'
+# Use Puma as the app server
+gem 'puma', '~> 4.1'
+# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
+gem 'webpacker', '~> 4.0'
+# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
+gem 'turbolinks', '~> 5'
+# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'jbuilder', '~> 2.7'
+# Use Redis adapter to run Action Cable in production
+# gem 'redis', '~> 4.0'
+# Use Active Model has_secure_password
+# gem 'bcrypt', '~> 3.1.7'
+
+# Use Active Storage variant
+# gem 'image_processing', '~> 1.2'
+
+# Reduces boot times through caching; required in config/boot.rb
+gem 'bootsnap', '>= 1.4.2', require: false
+
+group :development, :test do
+  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+end
+
+group :development do
+  # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
+  gem 'web-console', '>= 3.3.0'
+  gem 'rubocop'
```

Comment:
When doing the style guide check with Rubocop, I got this error:

![imagen](https://user-images.githubusercontent.com/12243331/101401224-a2f34080-3897-11eb-93a1-ad2d9484862e.png)

Add this line to the application's Gemfile:

`gem 'rubocop-packaging'`

It should be something like that in the Gemfile:

![imagen](https://user-images.githubusercontent.com/12243331/101401449-01b8ba00-3898-11eb-99ec-c90862140524.png)

Reply:
Using Rubocop allows us to identify style errors in the code:

For example, in this project, they already come by default:

![imagen](https://user-images.githubusercontent.com/12243331/101401952-b0f59100-3898-11eb-992a-33f531f14966.png)

And using Rubocop, they decrease to:

![imagen](https://user-images.githubusercontent.com/12243331/101402034-cec2f600-3898-11eb-8686-6a9f5505ba24.png)

To execute it, just place in the console, the command:

`rubocop -A
`